### DISCUSSION TITLE
Starship prompt

### DISCUSSION THREAD

**Discussion by VuiMuich** [nixpulvis/oursh#65] [Show and tell]
Just wanted to leave a not that it looks like initiating [starship](https://github.com/starship/starship) prompt with `source <("/use/bin/starship" zsh init --print-full-init)` works just fine.

**Comment by VuiMuich** [comment]
oh, never mind. I was stupid and didn't notice oursh crashed earlier and I was back in zsh all along..
You may as well delete this whole discussion thread.

Sorry for the inconvenience.

**Reply by nixpulvis** [reply]
What crashed, just curious.

There's **a lot** of unimplemented features here still.

**Reply by VuiMuich** [reply]
:thinking: it was some variation of the above command, that caused a panic, but I can't remember anymore, nor reproduce. Very sorry!

You may have a lot of unimplemented features, but I like this project a lot and am eagerly waiting for the day, when oursh will replace fish as my interactiv and zsh as my system shell!
